src/main/scala/io/flow/localization/rates/Localizer.scala

```diff
@@ -189,11 +217,14 @@ object LocalizerImpl {
 
 private[this] sealed trait KeyProvider {
   def getKey: String
+  def getUsaKey: String
 }
 
 private[this] case class CountryKey(country: String, itemNumber: String) extends KeyProvider {
```

Comment:
Let's remove `CountryKey` and `KeyProvider` to simply define a `def` in `object LocalizerImpl` to create the key?

Reply:
refactored to `def`s